specification/app/resource-manager/Microsoft.App/preview/2024-02-02-preview/SessionPools.json

```diff
@@ -0,0 +1,702 @@
+          "description": "Container App session pool resource specific properties",
+          "type": "object",
+          "properties": {
+            "environmentId": {
+              "description": "Resource ID of the session pool's environment.",
+              "type": "string",
+              "format": "arm-id",
+              "x-ms-arm-id-details": {
+                "allowedResources": [
+                  {
+                    "type": "Microsoft.App/managedEnvironments"
+                  }
+                ]
+              },
+              "x-ms-mutability": [
+                "create",
+                "read"
+              ]
+            },
+            "containerType": {
+              "description": "The container type of the sessions.",
+              "enum": [
+                "CustomContainer",
+                "PythonLTS"
+              ],
+              "type": "string",
+              "x-ms-enum": {
+                "name": "ContainerType",
+                "modelAsString": true
+              }
+            },
+            "poolManagementType": {
+              "description": "The pool management type of the session pool.",
+              "enum": [
+                "Manual",
+                "Dynamic"
+              ],
+              "type": "string",
+              "x-ms-enum": {
+                "name": "PoolManagementType",
+                "modelAsString": true
+              }
+            },
+            "nodeCount": {
+              "description": "The number of nodes the session pool is using.",
+              "type": "integer",
+              "format": "int32",
+              "readOnly": true
+            },
+            "scaleConfiguration": {
+              "description": "The scale configuration of the session pool.",
+              "$ref": "#/definitions/ScaleConfiguration"
+            },
+            "secrets": {
+              "description": "The secrets of the session pool.",
+              "type": "array",
+              "items": {
+                "$ref": "#/definitions/Secret"
+              },
+              "x-ms-identifiers": [
+                "name"
+              ]
+            },
+            "dynamicPoolConfiguration": {
+              "description": "The pool configuration if the poolManagementType is dynamic.",
+              "$ref": "#/definitions/DynamicPoolConfiguration"
+            },
+            "customContainerTemplate": {
+              "description": "The custom container configuration if the containerType is CustomContainer.",
+              "$ref": "#/definitions/CustomContainerTemplate"
+            },
+            "sessionNetworkConfiguration": {
+              "description": "The network configuration of the sessions in the session pool.",
+              "$ref": "#/definitions/SessionNetworkConfiguration"
+            },
+            "poolManagementEndpoint": {
+              "description": "The endpoint to manage the pool.",
+              "type": "string",
```

Comment:
>   "type": "string", [](http://example.com/codeflow?start=11&length=20)

Is this `"format": "uri"`
?

Reply:
Thanks, make sense, yes it should be uri, fixed